tools/ptree-overload/ptree-overload.c

```diff
@@ -56,7 +56,10 @@ dostuff(void *arg)
 	for (i = 0; i < opts.nptombs; i++) {
 		int rc;
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wformat-truncation" /* Ignored because 'k' + 6 digits + NUL byte = 8 */
```

Comment:
Format truncation. I think a uint64_t requires a 21 character buffer (1 for NUL), but in this case, we don't need that so we allocate a buffer of length 8 explained in the comment.